dev/Scroller/TestUI/ScrollerDynamicPage.xaml.cs

```diff
@@ -1299,6 +1309,24 @@ private void BtnQueueWidth_Click(object sender, RoutedEventArgs e)
             }
         }
 
+        private void BtnGetMaxWidth_Click(object sender, RoutedEventArgs e)
+        {
+            txtMaxWidth.Text = scroller.MaxWidth.ToString();
+        }
+
+        private void BtnSetMaxWidth_Click(object sender, RoutedEventArgs e)
+        {
+            try
+            {
+                scroller.MaxWidth = Convert.ToDouble(txtMaxWidth.Text);
```

Comment:
I was curious what the difference was between [this](https://docs.microsoft.com/en-us/dotnet/api/system.convert.todouble) and [Double.Parse](https://docs.microsoft.com/en-us/dotnet/api/system.double.parse). According to the docs... the Convert one is the same. Except that [Double.TryParse](https://docs.microsoft.com/en-us/dotnet/api/system.double.tryparse?view=netframework-4.7.2) exists. Did you want that or do you need this error data?

Reply:
Interesting. I'm just going to keep the current pattern as I like the 'in-your-face' error message when the conversion fails.